public/index.html

```diff
@@ -1,14 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+
+<head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="Find Your Immunization"
-      content="Find Your Immunization by Vaccine Hunters Canada finds you bookable vaccine appointments."
-    />
+    <meta name="Find Your Immunization" content="Find Your Immunization by Vaccine Hunters Canada finds you bookable vaccine appointments." />
+    <meta http-equiv="Content-Security-Policy" content="img-src 'self' 'unsafe-inline' data: https:; script-src *.twimg.com *.twitter.com *.google-analytics.com 'self' 'unsafe-inline'; style-src *.twimg.com *.twitter.com 'self' 'unsafe-inline'; default-src *.twitter.com 'self'; connect-src *.vaccinehunters.ca *.azurewebsites.net 'self';">
```

Comment:
Let's get rid of this - I was trying to override policy set by the header. We should have one or the other, and the header is preferred.